src/actions.cpp

```diff
@@ -715,11 +726,12 @@ void evm_contract::call_(const runtime_config& rc, intx::uint256 s, const bytes&
 
 void evm_contract::dispatch_tx(const runtime_config& rc, const transaction& tx) {
     if (_config->get_evm_version_and_maybe_promote() >= 1) {
-        process_tx(rc, get_self(), tx);
+        process_tx(rc, get_self(), tx, 0 /* min_inclusion_price */);
```

Comment:
Since we moved the assertio to the `pushtx` action body, we can call `process_tx` here with an empty optional `{}`